bedrock/firefox/templates/firefox/new/scene2_email.html

```diff
@@ -2,16 +2,12 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
+{% from "macros.html" import google_play_button with context %}
+
 {% add_lang_files "firefox/new/quantum" %}
 
 {% extends "firefox/base-protocol.html" %}
 
-{% block experiments %}
-  {% if switch('firefox_download_scene2_email_experiment', ['en-US']) %}
-    {{ js_bundle('firefox_download_scene2_email_experiment') }}
```

Comment:
Good spot - I think you should be able to remove this bundle definition and the experiment files too 👍